### PR TITLE
Adding Panasonic Connect Story

### PR DESCRIPTION
**Pull request by yuhattor**
This pull request mainly involves updates to the company listings on the stories pages of the website in two different languages: English, and Japanese. Specifically, a new entry for "Panasonic Connect" has been added to the list of companies in each language version of the site.

Here are the key changes:

* <a href="diffhunk://#diff-1e8776b0ec62524efd28bacf5c55fcb0aa3911460b32c40947ef490f4404988dR187-R189">`content/en/stories/_index.md`</a>: Added "Panasonic Connect" to the list of companies, along with a link to a case study and a quote about their adoption of Azure and GitHub for a cloud-native development culture.

* <a href="diffhunk://#diff-70173a5f4aa682e8545badf952cc19185252f8887338fbeb4e7a0588a3835affR187">`content/fr/stories/_index.md`</a>: Added the English version of the "Panasonic Connect" entry, with a link to the same case study.

* <a href="diffhunk://#diff-790d3394b087fb1d449526995a1206b2ec449e9f7c8c4ba12178b212611f5d43R187">`content/ja/stories/_index.md`</a>: Added the Japanese version of the "Panasonic Connect" entry, with a link to a Japanese case study and a quote about their transformation towards a cloud-native development culture.

### CODE REVIEW

**Comment by yuhattor** [comment]
[WIP] Confirmation from Panasonic Connect side:)
相原さん、こちら確認だけよろしくお願いいたします!!

File: content/ja/stories/_index.md [hunk 1]
```diff
@@ -184,6 +184,7 @@ aliases:
       {{< /company >}}
       {{< company name="Otto Group" image="/images/logos/ottogroup.png" article="https://github.com/customer-stories/otto-group" >}}
       {{< /company >}}
+       {{< company name="Panasonic Connect" image="/images/logos/panasonic-connect.png" article="https://customers.microsoft.com/ja-jp/story/1653141783429229154-panasonic-manufacturing-azure-ja-jp"  author_title="開発文化の変革に挑むパナソニック コネクトが Azure と GitHub を採用! 課題を乗り越えクラウドネイティブな次の景色を目指す" >}}
```
Comment (spier) [inline content/ja/stories/_index.md]:
```suggestion
       {{< company name="Panasonic Connect" image="/images/logos/panasonic-connect.png" article="https://customers.microsoft.com/ja-jp/story/1653141783429229154-panasonic-manufacturing-azure-ja-jp" author_title="開発文化の変革に挑むパナソニック コネクトが Azure と GitHub を採用! 課題を乗り越えクラウドネイティブな次の景色を目指す" >}}
```

File: content/fr/stories/_index.md [hunk 2]
```diff
@@ -184,6 +184,7 @@ aliases:
       {{< /company >}}
       {{< company name="Otto Group" image="/images/logos/ottogroup.png" article="https://github.com/customer-stories/otto-group" >}}
       {{< /company >}}
+       {{< company name="Panasonic Connect" image="/images/logos/panasonic-connect.png" article="https://customers.microsoft.com/en-us/story/1671022200964097988-connect-panasonic-discrete-manufacturing-azure-en-japan"  author_title="Panasonic Connect Adopts Azure and GitHub to Transform to a Cloud-Native Development Culture" >}}
```
Comment (spier) [inline content/fr/stories/_index.md]:
```suggestion
       {{< company name="Panasonic Connect" image="/images/logos/panasonic-connect.png" article="https://customers.microsoft.com/en-us/story/1671022200964097988-connect-panasonic-discrete-manufacturing-azure-en-japan" author_title="Panasonic Connect Adopts Azure and GitHub to Transform to a Cloud-Native Development Culture" >}}
```

File: content/en/stories/_index.md [hunk 3]
```diff
@@ -184,6 +184,9 @@ aliases:
       {{< /company >}}
       {{< company name="Otto Group" image="/images/logos/ottogroup.png" article="https://github.com/customer-stories/otto-group" >}}
       {{< /company >}}
+      {{< company name="Panasonic Connect" image="/images/logos/panasonic-connect.png" article="https://customers.microsoft.com/en-us/story/1671022200964097988-connect-panasonic-discrete-manufacturing-azure-en-japan"  author_title="Panasonic Connect Adopts Azure and GitHub to Transform to a Cloud-Native Development Culture" >}}
```
Comment (spier) [inline content/en/stories/_index.md]:
```suggestion
      {{< company name="Panasonic Connect" image="/images/logos/panasonic-connect.png" article="https://customers.microsoft.com/en-us/story/1671022200964097988-connect-panasonic-discrete-manufacturing-azure-en-japan" author_title="Panasonic Connect Adopts Azure and GitHub to Transform to a Cloud-Native Development Culture" >}}
```

File: content/en/stories/_index.md [hunk 4]
```diff
@@ -184,6 +184,9 @@ aliases:
       {{< /company >}}
       {{< company name="Otto Group" image="/images/logos/ottogroup.png" article="https://github.com/customer-stories/otto-group" >}}
       {{< /company >}}
+      {{< company name="Panasonic Connect" image="/images/logos/panasonic-connect.png" article="https://customers.microsoft.com/en-us/story/1671022200964097988-connect-panasonic-discrete-manufacturing-azure-en-japan" author_title="Panasonic Connect Adopts Azure and GitHub to Transform to a Cloud-Native Development Culture" >}}
+      The Process of InnerSource is actually the mindset of Collaboration.
```
Comment (spier) [inline content/en/stories/_index.md]:
```suggestion
```

I suspect this should be removed, as it is the same as for the company below?

**Review by spier — COMMENTED** [review]
LGTM overall. Left one comment where likely some text has to be removed.

File: content/en/stories/_index.md (hunk 4 above)
Comment (yuhattor) [reply content/en/stories/_index.md]: Yest definitely! Thank you!

**Comment by KoAihara** [comment]
We are pleased to have you add our logo to this website.
We approve this change.

**Comment by yuhattor** [comment]
Thank you @KoAihara san!

**Comment by yuhattor** [comment]
@InnerSourceCommons/website-maintainers Please approve this PR:)

**Review by andrea-kyurchiev — APPROVED** [review]

**Comment by andrea-kyurchiev** [comment]
Approved and merged but seems like https://www.entelgy.com/ have removed the Innersource acknowledgement from their website. Perhaps worth checking with them?

**Pull request merged**